docs/teleport/aks-getting-started.md

```diff
@@ -307,7 +306,51 @@ docker inspect teleport.azurecr.io/azure-vote-front:v1
   ]
 ```
 
-While you might consider flattening your images to one layer for fast mounting, while that would help, you may have contention on a single mount point. The purpose of the Teleport preview is to get further metrics on the usage to understand the art and science of image layers. The Teleport design does not require an image owner to make changes to use Teleport. Teleport works with your existing container images. However, with each technology, there are always optimizations that may be made based on the deployment target.
+### Comparing fewer layers
+
+To gauge the difference between layers and size, clone the [Azure-Samples/azure-voting-app-redis](https://github.com/Azure-Samples/azure-voting-app-redis) repo and build the image with the `--squash` flag.
+
+```bash
+docker build --force-rm --squash -t ${ACR}.azurecr.io/azure-vote-front:squashed .
+docker push ${ACR}.azurecr.io/azure-vote-front:squashed
+```
+
+- Change the `:tag` references in `azure-vote-shuttle.yaml` and `azure-vote-teleport.yaml` files to `:squashed`.
+- Follow the steps above to [recycle](#cleanup) the nodes, and [redeploy the two apps](#deploy-with-standard-pull-performance).
+
+The resulting times should reflect **31.7 seconds** for the standard docker pull/decompress and **1.9 seconds** for teleportation of a single layer. That's a reduction from 8.0 seconds to 1.9 seconds.
```

Comment:
Reduction from 8 seconds to 1.9? This is a little confusing shouldn't it be from 31.7 to 1.9?